composer.json

```diff
@@ -1,21 +1,8 @@
 {
-    "name": "elgg/community_theme",
-    "description": "Elgg community theme",
+    "name": "elgg/aalborg_theme",
```

Comment:
this will cause installation issues, since Packagist can't match the registered package with this name